zebra/zebra_nhg.c

```diff
@@ -3049,6 +3049,14 @@ static bool zebra_nhg_nexthop_compare(const struct nexthop *nhop,
 			if (IS_ZEBRA_DEBUG_NHG_DETAIL)
 				zlog_debug("%s: %pRN Old is not active going to the next one",
 					   __func__, rn);
+
```

Comment:
Could you add a comment here so we can capture the issue you're fixing - skipping nexthops that are present but inactive, for this specific path?

Reply:
sure done